packages/curvenote/src/cli/check.ts

```diff
@@ -2,21 +2,21 @@ import { Command } from 'commander';
 import { logCheckReport, runChecks } from '../check/runner.js';
 import type { ISession } from '../session/types.js';
 import { clirun } from './utils.js';
-import { abstractExists } from '../check/examples/abstractExists.js';
-import { abstractLength } from '../check/examples/abstractLength.js';
-import { availabilityExists } from '../check/examples/availabilityExists.js';
-import { linksResolve } from '../check/examples/linksResolve.js';
+import { submissionRuleChecks } from '@curvenote/check-implementations';
+import { build } from 'myst-cli';
 
 async function exampleChecks(session: ISession) {
+  await build(session, [], { all: true, checkLinks: true });
   const report = await runChecks(
     session,
     [
-      { id: 'abstract-exists' },
-      { id: 'abstract-length', max: '400' },
-      { id: 'availability-exists' },
-      { id: 'links-resolve' },
+      // { id: 'abstract-exists' },
+      // { id: 'abstract-length', max: '400' },
+      // { id: 'availability-exists' },
+      // { id: 'links-resolve' },
+      ...submissionRuleChecks,
```

Comment:
Could add this, just to visually differentiate between checks and check definitions... Doesn't actually change anything:

```suggestion
      ...submissionRuleChecks.map(({ id }) => {
        return { id };
      }),
```

(Also no need for the comments, I don't think.)